packages/terra-table/src/TableRow.jsx

```diff
@@ -53,6 +53,8 @@ const TableRow = ({
     customProps.className,
   ]);
 
+  customProps['aria-selected'] = isSelected ? true : false;
+
   const cloneChildren = cloneChildItems(children, height);
   return (
     <tr {...customProps} className={rowClassNames}>
```

Comment:
I know the linter threw a fit about this, but I too would prefer to not mutating customProps and just inlining the attribute like `<tr {...customProps} aria-selected={isSelected} className={rowClassNames}>`